### ISSUE TITLE
Discussie Eistypen

### ISSUE THREAD

**Issue by ElisabethKloren** [bimloket/contractspecificaties#48]
[n.reyngoud](https://hypothes.is/u/acct%3An.reyngoud%40hypothes.is)
[15 sep.](https://hypothes.is/a/M-KO3jT0Ee2iz4cdBO7raA)
[Public](https://hypothes.is/groups/__world__/public)
3.2.14 EisType
Ik mis hier een eistype waar je ontwerprandvoorwaarden in kwijt kunt. Bijvoorbeeld: 'De onderlinge afstand tussen de wildreflectoren op bermplanken dient maximaal 50 meter te bedragen.', 'Verharding met een open deklaag dient te zijn voorzien van asfaltlagen waarvan elke laag aan weerszijden minimaal 0,15m smaller is dan de daaronder gelegen laag', 'Bebording dient zo veel mogelijk gecombineerd te worden op één drager tot een maximum van drie borden.'. Wij classificeren die nu als 'Ontwerprandvoorwaarde', maar het zou fijn zijn daar een landelijk afgesproken type voor te kunnen gebruiken.

[tjmunster](https://hypothes.is/u/acct%3Atjmunster%40hypothes.is)
[29 sep.](https://hypothes.is/a/DAGACD__Ee2OKu9xj4P-kg)
ben ik mee eens. er worden (ontwerp)besluiten gemaakt. die niet toe te dichten zijn naar één of enkele aspecten. Dit belsuit kan je formuleren als eis voor de volgende fase(n). Dit soort eisen hebben een label nodig.

De eis dat de verbinding een brug dient te zijn. of dat de brug van het type basculebrug dient te zijn is heel lastig te hangen aan aspect of functie.

**Comment by ElisabethKloren** [comment]
Ik heb geantwoord in reviewdocument: Ik ga onderzoeken of hierover iets te vinden is in de leidraad se, en bespreken in gebruikerscommissie.

**Comment by ElisabethKloren** [comment]
@nreyngou gebruik jij zowel systeemeis als ontwerprandvoorwaarde?

**Comment by Kees-JensvdWel** [comment]
Hierbij antwoordt ik namens @nreyngou , Wij gebruiken bij de Provincie niet de term systeemeis maar Randvoorwaarde. Ik heb het ook getoetst bij RWS, daar wordt de term functionele eis en ontwerprandvoorwaarde toegepast. Het is een manier van definities en terminologie.

**Comment by ElisabethKloren** [comment]
Leidraad SE versie 3 noemt twee keer het woord ontwerprandoovrwaarde:

> Soms worden uit gewoonte of omwille van de efficiëntie eisen uit vorige projecten in de specificatie opgenomen. Hierdoor kunnen (naast tegenstrijdigheden) onnodige extra ontwerprandvoorwaarden worden opgenomen, waardoor (ongewenst) goede oplossingen worden uitgesloten. Wees hierop dus altijd alert

> De specificatie is integraal en loopt over alle disciplines en alle levensfasen van het systeem. Het bevat de bekende  onderdelen en alle eisen die men wil stellen. Daarbij is aandacht nodig voor: de omgevingsanalyse en alle belanghebbenden, contextobjecten,interne objecten of systeemdelen, functies, raakvlakkenen aspecteisen, prestatie-eisen en ontwerprandvoorwaarden. Het is goed om steeds opnieuw deze invalshoeken in beeld te brengen en ze in overeenstemming met elkaar te brengen

**Comment by ElisabethKloren** [comment]
[KeesJensWel](https://hypothes.is/u/acct%3AKeesJensWel%40hypothes.is)
[23 sep.](https://hypothes.is/a/THcXNDsXEe2UP28Nsogwtw)
[Public](https://hypothes.is/groups/__world__/public)
3.2.14 EisType
Ik mis hier nog: Ontwerprandvoorwaarde (kan ook gezien worden als systeemeis..) Aspect Uitvoering (Uitvoeringsgerelateerde eisen) Aspect Milieuhygiëne (dit is anders dan bv duurzaamheid) Externe raakvlakeis Interne raakvlakeis

**Comment by ElisabethKloren** [comment]
[tjmunster](https://hypothes.is/u/acct%3Atjmunster%40hypothes.is)
[29 sep.](https://hypothes.is/a/8GO1TD_9Ee2TceODhErVkA)
RAMSHE(EP) zijn veelvoorkomende aspecten. maar aspecten zijn niet limitatief opgesomt. een bijzonder project moet een aspectanalyse doen welke ontwerpaspecten relevant zijn. Dit kunnen hele andere thema's zijn. sociale veilighied, reproduceerbaarheid, weerstand (resilience) om zomaar iets te noemen. Ik zie dit rijtje als voorbeelden en ik denk niet dat dit document bedoelt is voor voorbeelden.

**Comment by ElisabethKloren** [comment]
Ik kom helaas tot de conclusie dat over Eistypen helaas nog te weinig consensus bestaat in de sector. Gelukkig hangen hier niet heel veel zaken aan vast zoals wel het geval is bij verificatiemethoden. 
Ik ga wel proberen, of bij het Provinciaal Contracten Buffet een standaard lijst kan worden afgesproken.

**Comment by ElisabethKloren** [comment]
@RiX012 check jij de taalbinding, ik denk dat die nu niet meer klopt omdat er geen verplichte lijst met eistypen is

**Comment by RiX012** [comment]
> @RiX012 check jij de taalbinding, ik denk dat die nu niet meer klopt omdat er geen verplichte lijst met eistypen is

Dus even ter check. We hebben nog steeds behoefte aan EisType. Dan is de taalbinding nog steeds naar `nen2660:requirementTopicType`. Alleen we hebben dus vanuit Contractspecificaties geen lijst meer te onderhouden?

**Comment by ElisabethKloren** [comment]
Ja, correct, men wil wel een eistype, maar geen verplichte lijst. Freedom for All

**Comment by RiX012** [comment]
Prima, istie weer weg!
